hwy/ops/rvv-inl.h

```diff
@@ -5278,6 +5278,12 @@ template <size_t kN, HWY_IF_LANES_GT(kN, 31)>
 constexpr unsigned MaxMaskBits() {
   return ~0u;
 }
+
+template <class D>
+constexpr int SufficientPow2ForMask() {
+  return std::max(
```

Comment:
How about HWY_MAX instead? Godbolt RVV sometimes has difficulty with std::.

Reply:
Thanks for pointing out this! I noticed there were no std:: in the codebase, but I wasn't sure what was being used as a substitute.